### DISCUSSION TITLE
Come funziona un termostato?

### DISCUSSION THREAD

**Discussion by esposimo** [indomus/forum#1017] [GENERALE - Discussioni generiche]
Brevemente..la domanda che mi sovviene è:

Ho una casa, N camere. Metto il termostato in soggiorno (es. Termostato Google nest), mi rileva la temperatura del soggiorno (e regolo di conseguenza) ovviamente.

Come regolo la temperatura in camera da letto? E' necessario metterne uno per camera?

**Comment by Scialla** [comment]
Se usi i termosifoni, devi avere le valvole termostatiche

**Comment by esposimo** [comment]
Perdona la domanda banale e da ignorante. Quale è l'alternativa al termosifone ?

**Reply by snakuzzo** [reply]
1. non ho capito la domanda
2. ti pregherei di rispondere sotto l'ultimo messaggio e non creando altri thread come hai fatto ora

Grazie

**Reply by Scialla** [reply]
riscaldamento a pavimento?

**Reply by riccionese** [reply]
fancoil ?

**Reply by riccionese** [reply]
radianti ad infrarossi ? :-)

**Reply by esposimo** [reply]
sto cercando di capire come regolare la temperatura in tutta la casa, per chiarirmi insomma le idee

Ho una caldaia
Ho dei termosifoni (es. 1 per ogni camera)

Compro un termostato (es. Google Nest Thermostat). 
Gli dico che voglio 20 gradi. 
Come fa lui, a regolare in tutta la casa ? Come fa a sapere la temperatura di ogni camera? C'è bisogno di altri componenti?

**Reply by snakuzzo** [reply]
infatti non può...solitamente sui termosifoni si applicano teste termostatiche (in questo caso smart) su valvole termostatiche

**Reply by esposimo** [reply]
quindi se ho le valvole sui termosifoni (per regolare la temp. in ogni camera) lo scopo del termostato quale è?

**Reply by Scialla** [reply]
Se hai riscaldamento autonomo di serve il termostato che accende e spegne la caldaia tramite relay, le valvole comunicano col termostato e non possono fare da sole. Le valvole chiudono il termosifone quando la temperatura è quella impostata

**Reply by esposimo** [reply]
Scusate ancora ma ho bisogno di ordinare un attimo le idee

Prendiamo in esame il termostato di google, ed escludiamo eventuali strumenti come hub etc.

Facciamo varie ipotesi su una casa con riscaldamento autonomo e termosifoni

Termosifoni senza valvole termostatiche
Termostato "connesso" alla caldaia.
Imposto i gradi sul termostato, es. 25 gradi.
Il termostato dirà alla caldaia di portare i termosifoni a 25°.
Non avendo le valvole, i termosifoni produrranno la stessa quantità di calore, ma essendo le stanze diverse (metratura, esposizione, etc) il risultato è che potresti avere temperature differenti in ambienti differenti (perchè magari una stanza che tende ad essere più fredda necessità di più calore)



Termosifoni con valvole termostatiche
Termostato connesso alla caldaia
Imposto i gradi sul termostato, es. 25 gradi.
Il termostato dirà alla caldaia di portare i termosifoni a 25°.
Avendo le valvole, i termosifoni "assorbiranno" più o meno gas/energia per arrivare alla temperatura desiderata, con il risultato di avere una temperatura globale in casa pari a quella scelta sul termostato, e con i termosifoni che consumano in maniera differente in base alle esigenze della camera in cui sono installati.
In questo caso quindi, il termostato funge solo da "selezionatore" della temperatura desiderata, comunicandolo alle valvole.

Perchè questa cosa non è menzionata , ad es., sulla scheda prodotto del termostato Nest ?

Dove mi sto perdendo?

**Reply by Scialla** [reply]
Perché Nest non produce valvole smart per i termo, ma solo il termostato.
Con gli altri puoi anche selezionare temperature diverse per le singole stanze, ad esempio i bagni più caldi delle altre stanze.
Io ho mia figlia che sta in camera sua a studiare e, di giorno, scaldo solo la sua stanza.
Puoi usare il termostato in una stanza e le valvole nelle altre; se metti valvole smart a tutti i termosifoni, se la caldaia non ha il ricircolo attivo, non riesce a sfogare e va in blocco quindi, senza ricircolo, un termo deve rimanere sempre aperto
